packages/xod-client-electron/src/shared/errorFormatter.js

```diff
@@ -1,5 +1,6 @@
 import R from 'ramda';
 import { ERROR_CODES as XFS_EC } from 'xod-fs';
+import { COMPILATION_ERRORS as XD_EC } from 'xod-deploy/dist/constants';
```

Comment:
Probably, a comment like "we import it like this because we need only the error codes and we don't mess with transpiling async/await" would be nice here :)

Reply:
Good idea 👍